pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+### Work Ticket
```

Comment:
**suggestion:** can we raise this in Engineer Huddle next week since we're not the only team that works in this repo? I assume everyone would be happy with it but I'd want to give the other devs a chance to provide feedback. I'd hold off on adding it in this PR and we can always add it separately.